tools/ci_build/build.py

```diff
@@ -133,6 +134,12 @@ def resolve_executable_path(command_or_path):
 def is_windows():
     return sys.platform.startswith("win")
 
+def is_linux():
+    return sys.platform.startswith("linux")
+
+def is_x64():
+    return platform.processor() == "x86_64"
```

Comment:
I think platform.processor() returns the real processor name. 
For me, it prints 'Intel64 Family 6 Model 63 Stepping 2, GenuineIntel'

I think what you want may be paltform.machine()

Documentation: https://docs.python.org/3/library/platform.html

Reply:
processor() gives x86_64 on linux.
But let's go with machine() then.